agenta-web/src/lib/helpers/analytics/AgPosthogProvider.tsx

```diff
@@ -26,12 +27,22 @@ const CustomPosthogProvider: CustomPosthogProviderType = ({children, config}) =>
                     if (process.env.NODE_ENV === "development") posthog.debug()
                 },
                 capture_pageview: false,
-                ...config,
+                ...(isDemo()
+                    ? {
+                          session_recording: {
```

Comment:
let's move this to a constants file in this analytics folder to keep things cleaner

Reply:
ahh, missed that 😞

Reply:
fixed